### DISCUSSION TITLE
ASM Hydrasynth Controller Presets

### DISCUSSION THREAD

**Discussion by rcaia** [helgoboss/helgobox#1337] [Show and tell]
Controller presets for ASM Hydrasynth.  I've done one for the Hydrasynth in CC mode and the other for NRPN mode.  Both need the Hydrasynth set to channel 2.  [If you need to change it's pretty easy to just search/replace the channel settings in the preset file with a text editor...]  Use case for me is strictly to control VST instruments.

The projections are setup to kinda mimic the Deluxe and also to keep screen real estate compact.  

ASM in CC mode : channel 2 [114 mappings]
[hydra-cc-ch2.json](https://github.com/user-attachments/files/17878615/hydra-cc-ch2.json)

ASM in NRPN mode : channel 2 [195 mappings... didn't do ALL the parameters possible...that might be just a bit much and not overly useful.]
[hydra-nrpn-ch2.json](https://github.com/user-attachments/files/17878616/hydra-nrpn-ch2.json)

Some of the NRPN parameters will not 'learn' correctly as they share NRPN numbers with other parameters by splitting the parameter into zones [like a synth keyboard zone split]... so here is a main compartment preset with all the mappings set to dummy targets.  

[hydra-template.json](https://github.com/user-attachments/files/17878636/hydra-template.json)

I load this template, save/as, then edit/delete/rename parameters as needed.  It has a mapping called "init asm" that does bank and patch change to F126 when active and back to A001 when inactive.  Feel free to edit the bank and patch or disable or delete this mapping if you want.  

The F126 patch is a 'ReaLearn' patch that I set up to have parameters available onscreen when you navigate around the Hydra so there isn't a need to enable or select options to show all the parameters I mapped.  [for example mutants are set to OSC sync so the 'window' parameter is availabe].  The dummy patch takes all the feedback and displays it [in ASM units of course] so you aren't changing things on a Hydra patch you might want to keep and then possibly saving over it by mistake.

**Comment by helgoboss** [comment]
Thanks! I will soon migrate unofficial presets to the Wiki so that they are at one place and can be not just created but also updated by all users. In this process, I will copy your files and descriptions there and notify you here.

**Comment by helgoboss** [comment]
@rcaia I copied your description to https://github.com/helgoboss/helgobox/wiki/ASM-Hydrasynth in order to make a list of all somehow-supported controllers available at a [single place](https://github.com/helgoboss/helgobox/wiki/ReaLearn-Controllers#presets-for-specific-controllers). Feel free to edit the page in case you want something changed.
